src/modules/i18n/components/locale-selector.vue

```diff
@@ -27,6 +28,6 @@ const localeOptions = computed(() =>
     v-model:value="locale"
     :options="localeOptions"
     placeholder="Select a language"
-    w-100px
+    w-120px
```

Comment:
Also mention this line since I found when initlizing zh-hant, the width seems not enough for it, so I add a little bit on pixels.